src/loader/LoadDataAction.cpp

```diff
@@ -209,18 +208,27 @@ Volume* LoadDataAction::run(bool convert)
   float minval[2];
   std::vector <cv::Mat> images;
   time_t posix_time;
+   std::cerr << "INIT Load Image 0"  << std::endl;
+   
   if (helper::ends_with_string(m_folder, "desc"))
   {
+     std::cerr << "INIT Load Image 1"  << std::endl;
+   
     FILE* pFile;
     pFile = fopen(m_folder.c_str(), "r");
+     std::cerr << "INIT Load Image 1.2"  << " pFile "<< m_folder.c_str()<<std::endl;
```

Comment:
why are you sending this to the error file? maybe you want to be using a logging that's a bit more configurable? I wouldn't expect info type of output in the error stream...